tulip/synth.py

```diff
@@ -980,28 +980,42 @@ def synthesize_many(specs, ts=None, ignore_init=None,
     assert isinstance(ts, dict)
     for name, t in ts.items():
         assert isinstance(t, transys.FiniteTransitionSystem)
-        ignore = name in ignore_init
-        bool_act = name in bool_actions
+        ignore = ignore_init.get(name) 
```

Comment:
The argument `ignore_init` is documented as a `set`:

https://github.com/tulip-control/tulip-control/blob/1ad5b12ce1f96253d898abb8485d4e0ca8d20d90/tulip/synth.py#L973

So regarding the motivation noted in the message of c1851c8be1659ad3caa88108a436a91816027909 (which uses `dict` instead of `set`):

```python
ts = dict(ts1=..., ts2=...)
ignore_init = set()
bool_actions = set()
```

The commit message presupposes an API change to make `ignore_init` and `bool_actions` dictionaries. This would be a separate change though (and a change in the docstring). Simple is better than complex ([PEP 20](https://www.python.org/dev/peps/pep-0020/)), so sets seem more suited to the purpose of these arguments.